### PR TITLE
Create Return Object page

### PR DESCRIPTION
**Pull request by jumpsplat120**
All over the wiki, "success" and "failure" objects are referenced, but it is not ever explained what they actually are. You can sort of piece it together by checking the `.ok()` page under scripts.lib, but instead I figured it would make more sense to just actually have a page that explained what it was and how it worked.

I've not created a page before, so the formatting is just based off what I observed from some of the other pages. As well, all the links on the wiki seem to be broken right now, so I'm not entirely sure if the link to the `ok` and `not_impl` pages have been formatted correctly, but I'm reasonably confident.

### CODE REVIEW

**Comment by seanmakesgames** [comment]
We merged https://github.com/comcode-org/hackmud_wiki/pull/498

Update your branch to get the test CI passing

**Comment by jumpsplat120** [comment]
`patch-4` and `patch-5` branches updated.

**Comment by seanmakesgames** [comment]
apparently wasn't enough. We got this one merged: https://github.com/comcode-org/hackmud_wiki/pull/499

try an update again this time and let's see if the tests pass / run correctly now. :)

File: docs/scripting/syntax/return_object [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Return Object
```
Comment (tukib) [inline docs/scripting/syntax/return_object:2]:
```suggestion
title: Return Object
alias:
  - success object
  - failure object
```

Might as well add these aliases here for future use

File: docs/scripting/syntax/return_object [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+---
+
+Built in function, and many user owned scripts return an object that has special formatting within the terminal. These special objects are often refered to as a ((%LSuccess%)) or ((%DFailure%)) object.
```
Comment (tukib) [inline docs/scripting/syntax/return_object:5]:
```suggestion
An object returned to the CLI from a script may be automatically formatted, depending on its contents. These types of objects are often referred to as ((%LSuccess%)) or ((%DFailure%)) objects.
```

I think this wording might be clearer for someone looking for a brief explanation for "what is this".

File: docs/scripting/syntax/return_object [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Syntax
+
+A ((%LSuccess%)) or ((%DFailure%)) object can only contain two keys. The `ok` key is required, and determines whether ((%LSuccess%)) or ((%DFailure%)) is displayed in the terminal. You may also attach a `msg` key, which will append that value, after being converted to a string.
+
+## Example
+
+```
```
Comment (tukib) [inline docs/scripting/syntax/return_object:13]:
```suggestion
```js
```

add js syntax highlighting

**Review by tukib — COMMENTED** [review]
This is great! I've added some suggestions, and you will need to rename the filename to `return_object.mdx` for it to be registered as wiki page.

**Review by tukib — APPROVED** [review]

**Pull request merged**